dist/scss/wds-components/_player.scss

```diff
@@ -0,0 +1,41 @@
+.wds-player-icon-play-tiny {
+	$size: 1.2 * 30px;
+
+	height: $size;
```

Comment:
you can use square mixin

Reply:
also SCSS support loop over map so you could generate these clasess
http://thesassway.com/intermediate/if-for-each-while